app/src/main/java/com/gmail/mxwild/newhabit/movieslist/FragmentMoviesList.kt

```diff
@@ -50,6 +50,16 @@ class FragmentMoviesList : Fragment() {
             viewModel.loadMoviesList(true)
         }
 
+        recycler.addOnScrollListener(object: RecyclerView.OnScrollListener() {
+            override fun onScrollStateChanged(recyclerView: RecyclerView, newState: Int) {
+                super.onScrollStateChanged(recyclerView, newState)
+
+                if (!recyclerView.canScrollVertically(1)) {
```

Comment:
What is 1?

Reply:
direction integers: -1 for up, 1 for down, 0 will always return false.